libpod/image/pull.go

```diff
@@ -310,9 +310,12 @@ func (ir *Runtime) doPullImage(ctx context.Context, sc *types.SystemContext, goa
 				return nil, err
 			}
 		}
-
-		_, err = cp.Image(ctx, policyContext, imageInfo.dstRef, imageInfo.srcRef, copyOptions)
-		if err != nil {
+		destRef := imageInfo.dstRef
```

Comment:
Nit: Why do you need to use variables?

Reply:
Gating tests complain about using `imageInfo` in the retry loop cause it's a variable from the outer loop.

Reply:
If that warning is about the loop variable changing in next iteration, affecting the closure, an `imageInfo := imageInfo` might work just as well.